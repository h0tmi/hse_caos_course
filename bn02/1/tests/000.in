Иван Иванов your_b64_encoded_letter
