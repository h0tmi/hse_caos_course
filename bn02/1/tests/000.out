Иван Иванов 100 баллов по АКОСу
